docs/reference/2-ods-api/how-to-guides/how-to-add-profiles-to-the-ed-fi-ods-api.md

```diff
@@ -6,17 +6,7 @@ definitions at run-time without rebuild and redeployment, it is preferred to
 configure profile definitions in the EdFi\_Admin database. Alternatively, you
 can define profile via embedded resource in a C# project as described in
 [Appendix
-A](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V72/pages/23301303/How+To+Add+Profiles+to+the+Ed-Fi+ODS+API#HowTo:AddProfilestotheEd-FiODS/API-AppendixA:AddingProfilesUsingtheVisualStudioProfilesProjectTemplate).
-
-- [How To: Add Profiles to the Ed-Fi ODS / API](#how-to-add-profiles-to-the-ed-fi-ods--api)
-  - [Step 1. Add Profiles via the EdFi\_Admin Database](#step-1-add-profiles-via-the-edfi_admin-database)
-  - [Step 2. Verify that the API Profiles feature is enabled](#step-2-verify-that-the-api-profiles-feature-is-enabled)
-  - [Step 3. Verify that the Profile endpoints are visible](#step-3-verify-that-the-profile-endpoints-are-visible)
-  - [Step 4. Confirm Profile Settings](#step-4-confirm-profile-settings)
-  - [Appendix A: Adding Profiles Using the Visual Studio Profiles Project Template](#appendix-a-adding-profiles-using-the-visual-studio-profiles-project-template)
-    - [Step 1. Create the Profiles Project](#step-1-create-the-profiles-project)
-    - [Step 2. Integrate Profiles into the Solution](#step-2-integrate-profiles-into-the-solution)
-    - [Step 3. Rebuild the Solution and Verify Changes](#step-3-rebuild-the-solution-and-verify-changes)
+A](#appendix-a-adding-profiles-using-the-visual-studio-profiles-project-template).
```

Comment:
Pro tip: Visual Studio Code will auto-complete local links after `#` but ONLY if the entire link is on one line.

For example, here we have

```
[Appendix 
A](....)
```

VS Code does not recognize that as a link, therefore it would not auto complete the hashtag for me.

But when I temporarily changed to

```
[Appendix A](# ...)
```

Then the autocomplete worked. After that, I was able to wrap the line again with the rewrap extension.